### DISCUSSION TITLE
원시 값 래퍼 객체에 대해 설명해보세요.

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#65] [원시 값과 객체의 비교]
- 원시 값 래퍼 객체에 대해 설명해보세요.

**Comment by pipisebastian** [comment]
> 원시 값 래퍼 객체에 대해 설명해보세요.

- 원시타입인 문자열을 객체처럼 접근하려고 할때, 자바스크립트 문자열을 임시로 객체로 만들어주어, 프로퍼티를 사용할 수 있게 해줍니다. 이때 만들어지는 객체를 래퍼객체라고 합니다.

**Comment by publdaze** [comment]
> - 원시 값 래퍼 객체에 대해 설명해보세요.
- 원시 값에 대해 객체처럼 프로퍼티나 메소드에 접근할 때 생성되는 임시 객체를 래퍼 객체라고 함
